backend/queue/queue.go

```diff
@@ -215,11 +201,6 @@ func (q *Queue) enqueueOps(backendIDs []string, value string) ([]clientv3.Cmp, [
 // Dequeue gets a value from the queue. It returns an error if the context
 // is cancelled, the deadline exceeded, or if the client encounters an error.
 func (q *Queue) Dequeue(ctx context.Context) (types.QueueItem, error) {
-	err := q.nackExpiredItems(ctx, q.itemTimeout)
```

Comment:
With this commit, items never expire from the queue? If that's the case, I believe ad-hocs checks would eventually be ran even in the event of a backend crashing/restarting and rejoining the queue with a new ID.

Reply:
They would not, because they would rejoin with a different ID, and therefore not receive the message.